cmd/oci-create-runtime-bundle/main.go

```diff
@@ -112,29 +102,26 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 	logrus.SetLevel(logLevel)
 
 	if v.version {
-		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		logrus.Infof("commit: %s", gitCommit)
+		logrus.Infof("spec: %s", specs.Version)
```

Comment:
OK, but I think it is easy and clear to use `fmt.Println()` directly here, WDYT?